packages/react-slider/src/components/Slider/Slider.test.tsx

```diff
@@ -455,19 +471,14 @@ describe('Slider', () => {
     });
 
     it('does not allow focus on disabled Slider', () => {
-      let sliderRef: any;
-
-      const SliderTestComponent = () => {
-        sliderRef = React.useRef(null);
-
-        return <Slider defaultValue={3} ref={sliderRef} data-testid="test" disabled />;
-      };
+      const sliderRef = React.createRef<HTMLDivElement>();
+      const inputRef = React.createRef<HTMLInputElement>();
 
-      render(<SliderTestComponent />);
+      render(<Slider ref={sliderRef} disabled input={{ ref: inputRef }} />);
 
       expect(document.activeElement).toEqual(document.body);
-
-      sliderRef.current.focus();
+      sliderRef?.current?.focus();
+      inputRef?.current?.focus();
```

Comment:
Put another expect call between the two to check at each step.
```suggestion
      sliderRef?.current?.focus();
      expect(document.activeElement).toEqual(document.body);
      inputRef?.current?.focus();
```